src/molecules/header-card/header-card.tsx

```diff
@@ -37,8 +37,20 @@ const HeaderCardContainer = styled.div`
 export const DefaultHeaderCard = ({ story, config, storyType }: HeaderCardProps) => {
   const { publisherConfig } = config;
 
+  const theme = useTheme();
+
   const { enableLastPublished, enableFirstPublished } = getDateSettings(config, storyType);
 
+  function authorCardRender(data) {
```

Comment:
can rename `authorCardRender` to `AuthorCard`

Reply:
rename to `authorCard`